### ISSUE TITLE
Segmentation label map

### ISSUE THREAD

**Issue by rose-jinyang** [SHI-Labs/OneFormer#42]
Hello
How are you?
Thanks for contributing to this project.
I want to get one segmentation map containing all the class labels rather than binary mask for each instance.
How can I get it?

**Comment by praeclarumjj3** [comment]
Hi, you can run a loop to aggregate all the predicted binary masks:

```python
# create an all-zeros mask
single_channel_mask = torch.zeros_like(image)

# loop through all instance masks
for cls_id, mask in zip(result.pred_classes, result.pred_masks):
    mask *= cls_id
    single_channel_mask = torch.max(single_channel_mask, mask)
```

**Comment by rose-jinyang** [comment]
Thanks for your help.
